sklearn/datasets/descr/diabetes.rst

```diff
@@ -21,11 +21,11 @@ quantitative measure of disease progression one year after baseline.
       - sex
       - bmi     body mass index
       - bp      average blood pressure
-      - s1      tc, T-Cells (a type of white blood cells)
+      - s1      tc, total serum cholesterol
       - s2      ldl, low-density lipoproteins
       - s3      hdl, high-density lipoproteins
-      - s4      tch, thyroid stimulating hormone
-      - s5      ltg, lamotrigine
+      - s4      tch, total cholesterol / HDL
+      - s5      ltg, possibly log of serum triglycerides level
```

Comment:
```suggestion
      - s5      ltg, log of serum triglycerides level
```

Reply:
Maybe we should add a warning to state that the reliability of the meaning of each feature is not as good as we would have liked because the documentation of the source dataset is not very explicit.

Reply:
I think a note in the API docs would be more appropriate

Reply:
I added a note to `sklearn.datasets.load_diabetes`